spec/suites/map/MapSpec.js

```diff
@@ -162,6 +162,31 @@ describe("Map", function () {
 		});
 	});
 
+	describe("#getBoundsZoom", function () {
+		var halfLength = 0.0001;
+		var bounds = [[-halfLength, -halfLength], [halfLength, halfLength]];
+		var padding = [100, 100];
+		var height = '400px';
+
+		it("returns high levels of zoom with small areas and big padding", function () {
+			var container = map.getContainer();
+			container.style.height = height;
+			document.body.appendChild(container);
+			expect(map.getBoundsZoom(bounds, false, padding)).to.be.equal(21);
+		});
+
+		it("returns multiples of zoomSnap when zoomSnap > 0 on any3d browsers", function () {
+			var container = map.getContainer();
+			container.style.height = height;
+			document.body.appendChild(container);
+			L.Browser.any3d = true;
+			map.options.zoomSnap = 5;
+			expect(map.getBoundsZoom(bounds, false, padding) % map.options.zoomSnap).to.be.equal(0);
```

Comment:
I compared the modulus to zero so the test would match the test description more, but I can see that this is basically accepting an infinite number of results so I've changed it to just compare the result to a number.